docs/dataset.md

```diff
@@ -116,111 +184,351 @@ This directory contains:
+1      102m__1__1.A__1.C    102m__A__C_c237197  train     c14  ...                             20                                 616.177293                        False                    True
+2  103m__1__1.A__1.C_1.D  103m__A__C_D_c252759  train     c14  ...                             16                                 616.177293                        False                   False
+3  104m__1__1.A__1.C_1.D  104m__A__C_D_c274687  train     c14  ...                             21                                 616.177293                        False                   False
+4  105m__1__1.A__1.C_1.D  105m__A__C_D_c221688  train     c14  ...                             20                                 616.177293                        False                   False
 ```
+:::
 
-where
-
-- `split_method`: the base method the split is based on:
-  - `ECOD`: split by ECOD domain
-  - ...
-- `similarity metric`: the similarity metric reported in the table:
-  - `pli_qcov`: ...
-  - `pocket_lddt`: ...
-  - ...
-- `learning_set`: the first split set of systems to be compared:
-  - `train`: training set
-  - `val`: validation set
-- `evaluation set`: the second split set of systems to be compared:
-  - `val`: validation set
-  - `test`: test set
+### Linked structures (`linked_structures/`)
+This directory contains the linked apo and predicted structures for PLINDER systems. These structures are intended to be used for augmenting the PLINDER dataset, eg. for flexible docking or pocket prediction purposes.
+The files are grouped into zipped subdirectories by using `two_char_code` of the system.
+Each unzipped subdirectory contains `pred` and `apo` subfolders that in turn contain folders named by `system_id`.
+Inside each `apo/{system_id}` and `pred/{system_id}` folder is another directory containing a superposed system: `{source_id}_{chain_id}/superposed.cif`, where `{source_id}` and `{chain_id}` for apo systems is `pdb_id` with a source chain identifier, and for predicted structures, `{source_id}` is `uniprot_id` used in AF2DB with a chain identifier set to `A`.
+
+
+### Linked systems (`links/`)
+This directory contains parquet files linking PLINDER systems to their apo and predicted structures in `linked_structures/`.
 
-:::{todo}
-- Fill missing methods etc. in list above
-:::
 
-:::{list-table} `plinder_<parameters>.parquet` columns
+:::{list-table} `{apo|pred}_links.parquet` columns
 :widths: 10 5 30
 :header-rows: 1
 
 *   - Name
     - Type
     - Description
-*   - system_id
+*   - reference_system_id
     - str
     - The PLINDER system ID
-*   - similarity
+*   - id
+    - str
+    - The PDB or AF2DB (for `apo` and `pred`, respectively) `{source_id}_{chain_id}` tag
+*   - pocket_fident
+    - float
+    - sequence identity for pocket residues
+*   - pocket_lddt
+    - float
+    - Local Distance Difference Test (lDDT) score for the pocket residues
```

Comment:
Local Distance Difference Test (lDDT) score for the pocket residue alpha carbons as returned by Foldseek